tests/scripts/test_scripts.py

```diff
@@ -601,7 +601,7 @@ def test_transfer_learning(tmpdir: str) -> None:
     Args:
         tmpdir: Temporary directory to save results to.
     """
-    tmpdir_path = pathlib.Path(tmpdir)
+    tmpdir_path = types.parse_path(tmpdir)
```

Comment:
What was wrong with just using `pathlib.Path` here given `tmpdir` is guaranteed to be a `str`?